nrf_security/doc/driver_config.rst

```diff
@@ -0,0 +1,344 @@
+.. _nrf_security_driver_config:
+
+Driver configurations and supported features
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section covers the configurations available when using PSA drivers.
+
+.. _nrf_security_drivers_config_multiple:
+
+Configuring multiple drivers
+****************************
+
+Multiple PSA drivers can be enabled at the same time, with added support for fine-grained control of which drivers implements support for cryptographic features.
+
+To enable a PSA driver, set the configurations according to the following table:
+
++---------------+------------------------------------+------------------------------------------------+
+| PSA driver    | Configurations                     | Notes                                          |
++================+===================================+================================================+
+| nrf_cc3xx     | :kconfig:`PSA_CRYPTO_DRIVER_CC3XX` | Only on nRF52840, nRF9160, and nRF5340 devices |
++---------------+------------------------------------+------------------------------------------------+
+| nrf_oberon    | :kconfig:`PSA_RYPTO_DRIVER_OBERON` |                                                |
++---------------+------------------------------------+------------------------------------------------+
+
+If multiple drivers are enabled, the first ordered item in this table takes precedence for an enabled cryptographic feature, unless the driver does not enable or support it.
+
+Enabling or disabling PSA driver specific configurations controls the support for a given algorithm, per driver.
+
+If a specific cryptographic feature is not supported by a PSA driver but the algorithm is configured to be used, then builtin Mbed TLS will be enabled to ensure the feature is available.
```

Comment:
"builtin Mbed TLS" needs to be defined.

It should be possible to understand that this is a Nordic fork of mbedtls that is supplied through west.

Reply:
We are inconsistent. Other places we call it "Orignal mbedtls"

Reply:
Fixed with more links to the description of what builtin means